### PR TITLE
added routers, users can now choose between solo and group play (UI Change)

### PR DESCRIPTION
**Pull request by jdicarlantonio**
This should be a better infrastructure for quickly setting up UI components

### CODE REVIEW

**Review by Arent128 — APPROVED** [review]

**Review by KeithsCreations — APPROVED** [review]
I'm learning how json works as I read along these changes.

**Comment by jdicarlantonio** [comment]
added file sharing and play/pause with multiple users

File: companion-app/src/Components/GroupPlay.js [hunk 1]
```diff
@@ -1,12 +1,257 @@
...
+        for (const i in groupUsers) {
+            let prefix = groupUsers[i];
+
+            Storage.put(prefix + '.json', groupPacket)
+            .then(result => console.log(result))
+            .catch(err => console.log(err));
+        }
+    }
+
+    setAudioState = () => {
```
Comment (blim012) [inline companion-app/src/Components/GroupPlay.js]: Toggle functionality seems strange in a "set" function

**Review by blim012 — APPROVED** [review]

File: companion-app/src/Components/GroupPlay.js (hunk 1 above)
Comment (jdicarlantonio) [reply companion-app/src/Components/GroupPlay.js]: changed naming

**Review by Arent128 — APPROVED** [review]

**Review by KeithsCreations — APPROVED** [review]

**Pull request merged**